rfcs/0003-radicle-daemon.rst

```diff
@@ -123,7 +123,23 @@ inputs received will be pinned.
   updated IPNS record. If this is successful, the daemon posts a
   message of type ``"new_input"`` with the same nonce to the
   machines's pubsub topic.
-  
+
+**Inaccessible computers and relayers**
+
+Many computers will be behind firewalls, or NATs that IPFS/pubsub cannot
+traverse. People who host writer chains on such computers can use
```

Comment:
The term “host writer chains” seems a bit opaque to me. I think it would be more descriptive to talk about the IPFS daemon that controls a machine’s IPNS ID